tracing/opentracing/client_interceptors.go

```diff
@@ -133,7 +138,18 @@ func newClientSpanFromContext(ctx context.Context, tracer opentracing.Tracer, fu
 	return opentracing.ContextWithSpan(ctxWithMetadata, clientSpan), clientSpan
 }
 
-func finishClientSpan(clientSpan opentracing.Span, err error) {
+func finishClientSpan(ctx context.Context, clientSpan opentracing.Span, err error) {
+	tags := grpc_ctxtags.Extract(ctx)
+	for k, v := range tags.Values() {
+		// Don't tag errors, log them instead.
+		if vErr, ok := v.(error); ok {
+			clientSpan.LogKV(k, vErr.Error())
+
+		} else {
+			clientSpan.SetTag(k, v)
```

Comment:
why are you using `LogKV` for one but `SetTag` for the others?

Reply:
This logic is copied from the `finishClientSpan` for server interceptors. The comment says that it does not tag errors...